packages/core/src/actions/index.ts

```diff
@@ -56,6 +56,14 @@ export const SET_LOCALIZED_UISCHEMAS: 'jsonforms/SET_LOCALIZED_UISCHEMAS' =
 export const ADD_DEFAULT_DATA: 'jsonforms/ADD_DEFAULT_DATA' = `jsonforms/ADD_DEFAULT_DATA`;
 export const REMOVE_DEFAULT_DATA: 'jsonforms/REMOVE_DEFAULT_DATA' = `jsonforms/REMOVE_DEFAULT_DATA`;
 
+export type ValidCoreActions =
```

Comment:
Shouldn't we just name them `CoreActions`?
```suggestion
export type CoreActions =
```